### PR TITLE
CLDSRV-179 - align with ARSN-128 breaking change

### PR DESCRIPTION
**Pull request by jbertran**
ARSN-128 changed the `open` operation from directly returning the DB handle to passing the DB handle as callback. This was making s3server crash.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -33,7 +33,7 @@ RUN cd /tmp \
     && rm -rf /tmp/Python-$PY_VERSION.tgz
 
 RUN yarn cache clean \
-    && yarn install --production --ignore-optional --ignore-engines  \
+    && yarn install --production --ignore-optional --ignore-engines --network-concurrency 1  \
```
Comment (francoisferrand) [inline Dockerfile:36]: looks like a different issue...

**Review by francoisferrand — APPROVED** [review]

**Review by welansari — APPROVED** [review]

**Review by miniscruff — DISMISSED** [review]
please hold on this one, the main file you changed is meant to be removed as part of my Armory work. ( I just woke up )

**Comment by francoisferrand** [comment]
> please hold on this one, the main file you changed is meant to be removed as part of my Armory work. ( I just woke up )

Unfortunately this is blocking multiple teams, so we don't have the leisure to wait (c.f. AN-130) ; sorry but I am thus going to overrule your request for change (but feel free to revert the change in the storage/armory part before moving forward with that change).

**Pull request merged**